dss-collection-nft/lib/go/test/types.go

```diff
@@ -0,0 +1,74 @@
+package test
+
+import (
+	"github.com/onflow/cadence"
+)
+
+type CollectionGroupData struct {
+	ID                     uint64
+	Name                   string
+	Open                   bool
+	TimeBound              bool
+	NFTIDInCollectionGroup map[uint64]bool
+}
+
+type NFTData struct {
+	ID                uint64
+	CollectionGroupID uint64
+	SerialNumber      uint64
+	CompletionDate    uint64
```

Comment:
type should be `UFix64`

Reply:
This is a go variable and we just have uint64. It whacks the decimals, which is okay.